contrib/craftscripts/clearblocks.js

```diff
@@ -0,0 +1,60 @@
+// Marks all blocks that have at least 3 exposed sides (adjacent to an air block)
+// and then deletes those blocks.
+// Based on jsa's MCEdit Filter clearBlocks.py
+
+importClass(Packages.com.sk89q.worldedit.world.block.BlockTypes)
+
+context.checkArgs(0, -1, "")
+
+var blocks = context.remember()
```

Comment:
To make this "easy to understand", this should probably be named `editSession`.

Reply:
Variable renamed